sktime/classification/distance_based/_proximity_forest.py

```diff
@@ -131,9 +131,14 @@ def distance(instance_a, instance_b, **params):
 def distance_predefined_params(distance_measure, **params):
     """Conduct distance measurement with a predefined set of parameters.
 
-    :param distance_measure: the distance measure to use
-    :param params: the parameters to use in the distance measure
-    :returns: a distance measure with no parameters
+    Parameters
+    ----------
+    distance_measure: the distance measure to use
```

Comment:
should include mention of type and valid values.
Not blocking, since it's been this way before, therefore out of scope for this PR.

Just in case you know what exactly goes here.

Reply:
Sure I'm going to add the valid distance measures for this function